tests/pipelines/kitctl/run.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: pipeline-template
+  namespace: tekton-pipelines
+spec:
+  params:
+  - name: name
+    value: "example"
+  - name: substrate-name
+    value: "YOUR_SUBSTRATE_NAME"
+  podTemplate:
+    nodeSelector:
+      kubernetes.io/arch: amd64
+  serviceAccountName: tekton-pipelines-executor
```

Comment:
i don't think this will give all you need to run this pipeline successfully. Have you tried running this template ? What was the output ?